pkgs/development/tools/cpm/default.nix

```diff
@@ -1,28 +1,26 @@
 { lib
 , stdenvNoCC
-, fetchFromGitHub
+, fetchurl
 }:
 
 stdenvNoCC.mkDerivation rec {
   pname = "cpm";
   version = "0.35.1";
 
-  src = fetchFromGitHub {
-    owner = "cpm-cmake";
-    repo = "CPM.cmake";
-    rev = "v${version}";
-    hash = "sha256-Oon/5iwkUUASsUDvde69iEwLe8/CAzwYKYsyzH5K+V0=";
+  src = fetchurl {
+    url = "https://github.com/cpm-cmake/CPM.cmake/releases/download/v${version}/CPM.cmake";
+    sha256 = "sha256-CMge+NpJRU+G+c+s0tb2EN8UG6E8FE90lIvcULggYXY=";
   };
 
+  dontUnpack = true;
   dontConfigure = true;
-
   dontBuild = true;
 
   installPhase = ''
     runHook preInstall
 
     mkdir -p ${placeholder "out"}/share/cpm/
-    cp ./cmake/CPM.cmake ${placeholder "out"}/share/cpm/
+    install -Dm644 ${src} ${placeholder "out"}/share/cpm/CPM.cmake
```

Comment:
No need to use `${placeholder "out"}` inside shell script, `$out` as environment variable will work fine.

Reply:
Thank you for your review!